tests/storage/cdi_import/test_import_http.py

```
@@ -131,13 +132,7 @@ def test_successful_import_image(
 )
 @pytest.mark.sno
 @pytest.mark.s390x
-def test_successful_import_secure_archive(
-    skip_block_volumemode_scope_module, internal_http_configmap, running_pod_with_dv_pvc
-):
-    """
-    Skip block volume mode - archive does not support block mode DVs,
-    https://github.com/kubevirt/containerized-data-importer/blob/main/doc/supported_operations.md
-    """
+def test_successful_import_secure_archive(internal_http_configmap, running_pod_with_dv_pvc):
```

Ruff (0.15.15)

[warning] Unused function argument: `internal_http_configmap`

(ARG001)